tools/build_bench.ts

```diff
@@ -0,0 +1,154 @@
+// Copyright 2018-2023 the Deno authors. All rights reserved. MIT license.
+
+import $ from "https://deno.land/x/dax/mod.ts";
+import { MultiProgressBar } from "https://deno.land/x/progress@v1.3.8/mod.ts";
```

Comment:
I believe `dax` has built-in support for progress bars: https://deno.land/x/dax@0.32.0#progress-indicator

Reply:
I'm not sure if it supports multiple progress bars, but let me check.